templates/verify_login.html

```diff
@@ -21,10 +21,12 @@
   <td valign="top">
     <h1 style="font-family: sans-serif; font-weight: normal; margin: 0 0 24px 0; text-align: center;">{{t "New sign-in to Firefox"}}</h1>
     <p class="primary" style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0 0 24px 0; text-align: center;">{{t "For added security, please confirm this sign-in to begin syncing with this device:"}}<br/><br/>
-    {{{ device }}}
-    {{#if ip }}<span style="display:block;margin:2px 0;">{{t "IP address: %(ip)s" }}</span>{{/if}}
-    {{#if location }}<span style="display:block;margin:2px 0;">{{ location }}</span>{{/if}}
-    {{#if timestamp }}<span style="display:block;margin:2px 0;">{{ timestamp }}</span>{{/if}}
+    <span style="display:block;margin:2px 0;">{{{ device }}}</span>
+{{#if location }}<span style="display:block;margin:2px 0;">{{ location }}</span>{{/if}}
+{{#if ip }}<span style="display:block;margin:2px 0;">{{t "IP address: %(ip)s" }}</span>{{/if}}
+{{#if timestamp }}<span style="display:block;margin:2px 0;">{{ timestamp }}</span>{{/if}}
```

Comment:
All the .html files in `templates` are generated using `grunt templates` from the nunjucks templates in `partials`. He. We have a double template system, nunjucks templates are used to create handlebars templates that are rendered by the mailer.